app/model/ApplicationStatus.scala

```diff
@@ -24,6 +24,8 @@ object ApplicationStatus extends Enumeration {
   val WITHDRAWN, CREATED, IN_PROGRESS, SUBMITTED = Value
   val PHASE1_TESTS = Value
 
+  implicit def toString(applicationStatus: ApplicationStatus): String = applicationStatus.toString
```

Comment:
They do, but it's not implicit and it was mostly so I didn't have to keep calling ```toString``` on variables